src/base/dof_map_constraints.C

```diff
@@ -431,6 +431,9 @@ private:
         std::vector<bool> is_boundary_node(elem->n_nodes(), false),
           is_boundary_edge(elem->n_edges(), false),
           is_boundary_side(elem->n_sides(), false);
+
+        // We also maintain a separate list of nodeset-based boundary nodes
+        std::vector<bool> is_boundary_nodeset(elem->n_nodes(), false);
```

Comment:
> OK, but I just made it consistent with the preceding lines.

I know, just giving @roystgnr a hard time.

> P.S. If you do want to change all four lines, perhaps you could do that in a separate commit (outside of this PR)?

Agreed, sorry didn't mean to make it sound like you were supposed to do that.